source/common/CMakeLists.txt

```diff
@@ -9,6 +9,12 @@ if (UR_BUILD_ADAPTER_L0 OR UR_BUILD_ADAPTER_L0_V2)
     set(UMF_LEVEL_ZERO_INCLUDE_DIR "${LEVEL_ZERO_INCLUDE_DIR}" CACHE INTERNAL "Level Zero headers")
 endif()
 
+if(UR_BUILD_ADAPTER_CUDA)
+    include(FetchCUDA)
+    set(UMF_BUILD_CUDA_PROVIDER ON CACHE INTERNAL "Build UMF CUDA provider")
+    set(UMF_CUDA_INCLUDE_DIR "${CUDA_INCLUDE_DIR}" CACHE INTERNAL "CUDA headers")
+endif()
```

Comment:
`else UMF_BUILD_CUDA_PROVIDER OFF` as it is `ON` by default in UMF.

Reply:
Done